downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1561 @@
+        SubscriptionTemporaryRedirect|BadSubscriptionError|InternalSubscriptionError {
+        // implement action to execute on new subscription
+        }
+
+    remote function onSubscriptionValidation(Subscription msg) returns SubscriptionDeniedError? {
+        // implement action to execute on subscription validation
+        }
+
+    remote function onSubscriptionIntentVerified(VerifiedSubscription msg) {
+        // implement action to execute on subscription intent verification
+        }
+
+    remote function onUnsubscription(Unsubscription msg) returns UnsubscriptionAccepted|BadUnsubscriptionError|
+        InternalUnsubscriptionError {
+        // implement action to execute on unsubscription
+        }
+
+    remote function onUnsubscriptionValidation(Unsubscription msg) returns UnsubscriptionDeniedError? {
+        // implement action to execute on unsubscription validation
+        }
+
+    remote function onUnsubscriptionIntentVerified(VerifiedUnsubscription msg) {
+        // implement action to execute on unsubscription intent verification
+        }
+    }    
+    ```
+
+**Hub Client Implementation**
+
+- The `websubhub:Client` is introduced to distribute the updated content to subscribers.
+- The example below is a sample use-case of the WebSub Hub Client.
+
+```ballerina
+    service / websubhubon hubListener {
+    remote function onSubscriptionIntentVerified(websubhub:VerifiedSubscription msg) {
+
+            // you can pass client config if you want 
+            // say maybe retry config
+            websub:HubClient hubclient = new (msg);
+            check start notifySubscriber(hubclient);
+        }
+
+        function notifySubscriber(websubhub:HubClient hubclient) returns error? {
+            while (true) {
+                // fetch the message from MB
+                check hubclient->notifyContentDistribution({content: "This is sample content delivery"});
+            }
+        }
+    }
+```
+
+**Publisher Implementation**
+
+The `websub:PublisherClient` is moved to the `websubhub:PublisherClient`.
+
+##### Removed Modules
+
+###### Config
+
+The configuration use cases are now covered under the `configurable` language feature. 
+
+###### Math
+
+The APIs related to random number generation were moved to the new `random` module. The rest of the APIs have replacements in the `lang.float` and `lang.int` packages.
+
+###### Stringutils
+
+The regex-related APIs that were supported by this module have been moved to the new `regex` module. The rest of the APIs have replacements in the langlib packages.
+
+###### Socket
+
+The `socket` module was removed and got replaced by the `TCP` and `UDP` modules.
+
+#### Observability
+
+- A new extension model, which separates each extension into a separate module is introduced.
+    - Observability is not included into the final JAR by default. It can be added with the following configuration in the `Ballerina.toml`
```

Comment:
When we create a new project, the `Ballerina.toml` file contains the following entry, right? So is it correct to say it is not included by default? @nadundesilva

Reply:
Yes the entry is included in the `Ballerina.toml` if we create using `ballerina new` command. However, if the entry is not present, the default behavior is to not include Observability.

Reply:
What if we say 
```suggestion
    - Observability can be included in the final JAR by adding the following configuration in the `Ballerina.toml` file.
```